README.md

```diff
@@ -10,22 +10,164 @@ Inside the Arsenal Library you have different options to make amazing and quick
 
 Transitions:
--Fade
--Slide
--Explode
--Slide Explosion (Custom from Library)
+
+- Fade
+- Slide
+- Explode
+- Slide Explosion (Custom from Library)
 
 ###Animations
 
 Resource Animations:
--Fade In
--Fade Out
--Slide Left
--Slide Right
--Rotate
--Custom (Application Resource)
 
-##Usage
+- Fade In
+- Fade Out
+- Slide Left
+- Slide Right
+- Rotate
+- Custom (Application Resource)
+
+##In Use
+
+For Window Transitions simply call:
+```java
+  //HERE WE ARE SETTING THE ENTER TRANSITION
+  AnimationArsenal.setEnterTransition(getWindow(), AnimationArsenal
+          .getFadeTransition(new Transition.TransitionListener()
+          {
+              @Override
+              public void onTransitionStart(Transition transition)
+              {
+              }
+
+              @Override
+              public void onTransitionEnd(Transition transition)
+              {
+                  //IN HERE YOU CAN START OTHER ANIMATIONS AFTER THE ACTIVITY FINISHES THE 
+                  // ENTER TRANSITION
+                  AnimationArsenal.circularReveal(myView,
+                          getApplicationContext(), AnimationArsenal.RevealGravity.CENTER);
+              }
+
+              @Override
+              public void onTransitionCancel(Transition transition)
+              {
+              }
+
+              @Override
+              public void onTransitionPause(Transition transition)
+              {
+              }
+
+              @Override
+              public void onTransitionResume(Transition transition)
+              {
+              }
+          }, Fade.IN));
+
+
+  //AS WELL YOU CAN HAVE TOTAL CONTROL OF THE EXIT TRANSITION
+  AnimationArsenal.setExitTransition(getWindow(), AnimationArsenal
+          .getExplodeTransition(new Transition.TransitionListener()
+          {
+              @Override
+              public void onTransitionStart(Transition transition)
+              {
+                  //IN HERE YOU CAN START OTHER ANIMATIONS AT THE BEGINNING OF EXIT
+                  // TRANSITION IN THE ACTIVITY
+                  AnimationArsenal.playAnimationFadeOut(getApplicationContext(), myView, 700,
+                          null);
+              }
+
+              @Override
+              public void onTransitionEnd(Transition transition)
+              {
+              }
+
+              @Override
+              public void onTransitionCancel(Transition transition)
+              {
+              }
+
+              @Override
+              public void onTransitionPause(Transition transition)
+              {
+              }
+
+              @Override
+              public void onTransitionResume(Transition transition)
+              {
+              }
+          }, 700));
+```
+Animation Resource Animations
+
+You can play Android Resource Animations as well as custom made animations from your application project.
+```java
+  //ANDROID RESOURCE EXAMPLE
+  AnimationArsenal.playAnimationScale(myView, 700, 0, 1,
```

Comment:
I am wondering if we can find a better name than `playAnimationScale`, probably `doScaleAnimation` ?